addons/payment/controllers/post_processing.py

```diff
@@ -44,8 +45,8 @@ def poll_status(self, **_kwargs):
         monitored_tx = request.env['payment.transaction'].sudo().browse(
             self.get_monitored_transaction_id()
         ).exists()
-        if not monitored_tx:  # The session might have expired, or the tx has never existed.
-            raise Exception('tx_not_found')
+        if not monitored_tx:
+            raise ValidationError(_('The session might have expired, or the tx is not found'))
 
```

Comment:
raise ValidationError(_('The session might have expired, or the transaction is not found'))

And add it in pot file